docs/installing-environment-modules-without-root-permissions.md

```diff
@@ -1,139 +1,116 @@
+    higher, since the environment modules package is not compatible with
+    those Tcl versions.
 
-    See <https://github.com/easybuilders/easybuild-docs> for more information.
+1. Unpack the Tcl source tarball:
 
-```rst
+    ``` shell
+    tar xfvz tcl8.5.15-src.tar.gz
+    ```
 
-.. _installing_env_mod_c:
+1. Pick a location where you will install Tcl. It should be a directory
+    you have write permissions on. My suggestion would be to use
+    something like `$HOME/.local/Tcl`.
 
-Installing environment modules without root permissions
-=======================================================
+1. Go to the `unix` subdirectory of the unpacked Tcl directory, and run
+    the `configure` script using the `--prefix` option:
 
-This short guide will explain how to install the standard environment modules Tcl/C
-software package without root permissions on a Linux or Mac OS X system,
-together with Tcl on which it depends.
+    ``` shell
+    cd tcl8.5.15/unix
+    ./configure --prefix=$HOME/.local/Tcl
+    ```
 
-Tcl
-~~~
+    If you're building Tcl and environment modules on Mac, you should
+    run `configure` in the `tcl8.5.15/macosx` directory instead.
 
-#. Go to `<http://www.tcl.tk>`_ and download the latest Tcl sources.
-   At the time of writing, the latest available Tcl version was 8.5.15,
-   which can be downloaded from
-   `here <http://prdownloads.sourceforge.net/tcl/tcl8.5.15-src.tar.gz>`_.
-   The remainder of these commands will assume Tcl v8.5.15 is being
-   installed, you may need to adjust them accordingly.
-   **Note**: Stick to Tcl v8.5.x, don’t consider using the more recent
-   v8.6.x or higher, since the environment modules package is not
-   compatible with those Tcl versions.
+1. Next, build Tcl using the `make` command. If the system you are
+    building on has multiple cores, running make in parallel will speed
+    up the build. Just use the `-j` option, and pass it a degree of
+    parallelism (just use the number of cores your system has
+    available), e.g.:
 
-#. Unpack the Tcl source tarball:
+    ``` shell
+    make -j 4
+    ```
 
-   .. code:: bash
+1. The final step consists of installing Tcl to the directory specified
+    in step 4. To do this, simply run:
 
-       tar xfvz tcl8.5.15-src.tar.gz
+    ``` shell
+    make install
+    ```
 
-#. Pick a location where you will install Tcl. It should be a directory
-   you have write permissions on.
-   My suggestion would be to use something like ``$HOME/.local/Tcl``.
+**All done!** Now you are ready to build the environment modules
+package, which requires Tcl.
 
-#. Go to the ``unix`` subdirectory of the unpacked Tcl directory, and
-   run the ``configure`` script using the ``--prefix`` option:
+## Environment Modules
 
-   .. code:: bash
+1. Download the latest source tarball for the environment modules tools
+    from <https://modules.sourceforge.net>. At the time of writing, the
+    latest available version is 3.2.10 which can be downloaded [from
```

Comment:
should this be updated to reflect the current "as of writing" version?

Reply:
We'll handle any such updates separately.